.github/workflows/container-startup-validation.yml

```
@@ -0,0 +1,335 @@
+          if [[ "${{ steps.changes.outputs.bunkbot }}" == "true" ]] || \
+             [[ "${{ steps.changes.outputs.covabot }}" == "true" ]] || \
+             [[ "${{ steps.changes.outputs.djcova }}" == "true" ]] || \
+             [[ "${{ steps.changes.outputs.starbunk-dnd }}" == "true" ]] || \
+             [[ "${{ steps.changes.outputs.snowbunk }}" == "true" ]] || \
+             [[ "${{ steps.changes.outputs.shared }}" == "true" ]] || \
+             [[ "${{ steps.changes.outputs.container-validation }}" == "true" ]]; then
+            echo "any-container=true" >> $GITHUB_OUTPUT
+          else
+            echo "any-container=false" >> $GITHUB_OUTPUT
+          fi
+    steps:
```

actionlint (1.7.7)

key "steps" is duplicated in "detect-changes" job. previously defined at line:53,col:5

(syntax-check)